packages/os/bootstrap-commands-toml

```diff
@@ -0,0 +1,22 @@
+[required-extensions]
+bootstrap-commands= "v1"
+std = { version = "v1", helpers = ["if_not_null", "join_nested_array"]}
++++
+{{#if_not_null settings.bootstrap-commands}}
+{{#each settings.bootstrap-commands}}
+[bootstrap-commands."{{@key}}"]
+{{#if_not_null this.commands}}
+commands = [
+{{#each this.commands}}
+    [{{#each this}}'''{{{ this }}}''',{{/each}}],
+{{/each}}
```

Comment:
I think this will work for most cases, unless a user uses `'''` for some reason. That would be unusual but not inconceivable.

There are two other ideas I had:
* Write a helper which encodes an input `serde_json::Value` as an inline toml value using [the toml crate](https://docs.rs/toml/).
* Embrace HTML escaping, using "double stache", and then have the `bootstrap-commands` binary decode the HTML-escaped input using [the `html-escape` crate](https://crates.io/crates/html-escape)